### PR TITLE
Incorporate cli behaviour in the capella base image

### PR DESCRIPTION
**Pull request by amolenaar**
See #146.

This PR only migrates the cli containers.

### CODE REVIEW

File: docs/docs/capella/cli.md
Comment (MoritzWeber0) [inline docs/docs/capella/cli.md:1]: It doesn't make sense to have two documentation pages or `capella/base`. Please move the docs to the `capella/base` documentation page. Additionally, add a deprecation note and point to the `capella/base` documentation.

Comment (MoritzWeber0) [inline docs/docs/capella/cli.md:1]: The `docs/index.md` still contains references to `capella/cli`. Please remove them.

**Comment by MoritzWeber0** [comment]
Please use conventional commits and run `pre-commit install` to activate the `pre-commit` hooks.

**Comment by amolenaar** [comment]
updated docs and commit messages.

File: docs/docs/capella/base.md [hunk 1]
```diff
@@ -12,6 +12,15 @@
 The Capella base image installs a selected Capella client version. The Capella client can be downloaded and can optionally be customised prior to building the Docker
 image or can be downloaded automatically in the Docker image.
 
+The images are meant to have a containerised Capella (with or without a
+Team for Capella client) that can be run headless (as command line interface).
+
+<!-- prettier-ignore -->
+!!! info
+    The functionalirt for running capella as a commanline app used to be part of
```
Comment (MoritzWeber0) [inline docs/docs/capella/base.md:20]:
```suggestion
    The functionality for running capella as a command-line app used to be part of
```

**Review by MoritzWeber0 — CHANGES_REQUESTED** [review]
I still see one left-over of `capella/cli` in the mermaid graph of the `index.md`. Otherwise, it looks good.

**Comment by amolenaar** [comment]
> I still see one left-over of `capella/cli` in the mermaid graph of the `index.md`. Otherwise, it looks good.

I considered we should leave it in, so search engines can still land on that page. 404's are bad for site score.

**Comment by MoritzWeber0** [comment]
> > I still see one left-over of `capella/cli` in the mermaid graph of the `index.md`. Otherwise, it looks good.
> 
> I considered we should leave it in, so search engines can still land on that page. 404's are bad for site score.

I'm just referring to the mermaid graph, not the page. The removal in the mermaid graph should not lead to 404's.

**Comment by amolenaar** [comment]
> I'm just referring to the mermaid graph, not the page. The removal in the mermaid graph should not lead to 404's.

Fixed.

**Review by MoritzWeber0 — APPROVED** [review]

**Pull request merged**